Cargo.toml

```diff
@@ -71,8 +71,8 @@ holochain_wasmer_guest = "=0.0.92"
 holochain_wasmer_host = "=0.0.92"
 mockall = "0.11"
 must_future = "0.1"
-lair_keystore = "0.4.0"
-lair_keystore_api = "=0.4.0"
+lair_keystore = { version = "0.4.1", default-features = false }
```

Comment:
This is important, the features in the workspace are additive with the features specified in the project `Cargo.toml`s. Subtly that includes the default features. So specifying `default-features = false` in `crates/holochain_keystore/Cargo.toml` does nothing because it's combined with this which turns it back on.

Not that we should regularly be turning encryption off for lair but without this, we lose access to options like linking against system crypto libraries rather than building from source. Still something we don't want to do often but since those are options, I would prefer not to break them here.

My source for the information above -> https://doc.rust-lang.org/cargo/reference/workspaces.html#the-dependencies-table